lib/src/view.rs

```diff
@@ -204,6 +204,7 @@ impl View {
     }
 
     pub fn remove_remote_branch(&mut self, name: &str, remote_name: &str) {
+        self.remove_git_ref(&format!("refs/remotes/{remote_name}/{name}"));
```

Comment:
Should `remove`/`set_git_ref()` be done transparently by view?

"local jj -> git" is handled by `export_refs`, and "local/remote git -> jj" is handled by `import_refs` afaik. I'm not sure if "remote jj -> git" is a job of the view API.

Reply:
I think you're right. Those changes can be moved to `commands/git.rs` where we explicitly manipulate git refs. Done.